spanner/ddl/ast.go

```diff
@@ -20,110 +20,67 @@
 // https://cloud.google.com/spanner/docs/data-definition-language.
 // Before defining each type, we give the snippet from this definition.
 //
-// We use go interface types to preserve the structural constraints of
-// Spanner DDL. For example, suppose ScalarType could be either
-// an INT or a STRING with a length specifier:
-//    ScalarType := INT | STRING( length )
-// then we use a go interface type to encode ScalarType:
-//    type ScalarType interface {
-//        PrintScalarType() string  // To print ScalarTypes types.
-//    }
-// and struct types for each case:
-//    type Int struct { }
-//    type String struct { length int64 }
-// and finally, we define functions:
-//    func (i Int) PrintScalarType() { ... }
-//    func (s String) PrintScalarType() { .. }
-//
-// The net result is that the only way to build a ScalarType
-// is using Int or String.
+// Package ddl provides a generic representation of Spanner DDL.
```

Comment:
Redundant comment -- looks like a repeat of line 15?

Reply:
Removed it.